ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -690,19 +690,15 @@ periodics:
         chmod ug+x /tmp/bin/oc
 
         # annotate the information about this job so that the release controller adds it to the upgrade graph
-        oc --kubeconfig /etc/appci/sa.release-bot.app.ci.config annotate pj/${PROW_JOB_ID} "release.openshift.io/from-tag=${from}" "release.openshift.io/tag=${to}" "release.openshift.io/source=ocp/$(BRANCH)"
+        oc --kubeconfig /etc/appci/sa.release-bot.app.ci.config annotate pj/${PROW_JOB_ID} "release.openshift.io/from-tag=${from}" "release.openshift.io/tag=${to}" "release.openshift.io/source=ocp/4.1"
```

Comment:
Noticing these, do we even need to keep the 4.1 and 4.2 periodics? Aren't they EOL'd already?

Reply:
Orthogonal :)